CMakeLists.txt

```
@@ -31,7 +31,7 @@ project(microxrcedds VERSION "2.1.0")
 
 set(_client_tag develop)
 set(_client_version 2.1.0)
-set(_agent_tag develop)
+set(_agent_tag fix/delete_entities)
```

```suggestion
set(_agent_tag feature/can_flag)
```

Reuse to test https://github.com/eProsima/Micro-XRCE-DDS-Agent/pull/294